cmd/agent.go

```diff
@@ -43,11 +44,11 @@ func init() {
 		"",
 		"Authorization token. If used, it will override the authorization token in the configuration file.",
 	)
-	agentCmd.PersistentFlags().UintVarP(
+	agentCmd.PersistentFlags().DurationVarP(
 		&agent.Period,
 		"period",
 		"p",
-		3600,
+		time.Hour,
 		"Time between scans, in seconds.",
```

Comment:
Ah, this has the same issue in the help now.

Reply:
Good spot!